### PR TITLE
Update admx info

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1564

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 1]
```diff
@@ -40,23 +40,23 @@ The following policy settings can be configured for UE-V.
 </thead>
 <tbody>
 <tr class="odd">
+<td align="left"><p>Configure Sync Method</p></td>
+<td align="left"><p>Computers and Users</p></td>
+<td align="left"><p>By using this Group Policy setting, you can configure whether UE-V uses the sync provider feature. This policy setting also lets you enable notification to appear when the import of user settings is delayed.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:45]:
```suggestion
<td align="left"><p>By using this Group Policy setting, you can configure whether User Experience Virtualization (UE-V) uses the sync provider feature. This policy setting also lets you enable a notification to appear when the import of user settings is delayed.</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 2]
```diff
@@ -40,23 +40,23 @@ The following policy settings can be configured for UE-V.
...
+<td align="left"><p>Enable this setting to configure the UE-V Agent not to use the sync provider, or to use external synchronization engine.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:46]:
```suggestion
<td align="left"><p>Enable this setting to configure the UE-V Agent not to use the sync provider, or to use the external synchronization engine.</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 3]
```diff
@@ -65,11 +65,10 @@ The following policy settings can be configured for UE-V.
 <td align="left"><p>The default is enabled.</p></td>
 </tr>
 <tr class="odd">
-<td align="left"><p>Roam Windows settings</p></td>
+<td align="left"><p>Ping the settings storage location before sync</p></td>
 <td align="left"><p>Computers and Users</p></td>
-<td align="left"><p>This Group Policy setting configures the synchronization of Windows settings.</p></td>
-<td align="left"><p>Select which Windows settings synchronize between computers.</p>
-<p>By default, Windows themes, desktop settings, and Ease of Access settings synchronize settings between computers of the same operating system version.</p></td>
+<td align="left"><p>This policy setting allows to configure the UE-V sync provider to ping the settings storage path before attempting to sync settings, to verify the connection.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:70]:
```suggestion
<td align="left"><p>This policy setting allows configuration of the UE-V sync provider to ping the settings storage path before attempting to sync settings, to verify the connection.</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 4]
```diff
@@ -110,17 +109,30 @@ The following policy settings can be configured for UE-V.
...
+<td align="left"><p>Synchronize Windows settings </p></td>
+<td align="left"><p>Computers and Users</p></td>
+<td align="left"><p>This Group Policy setting configures the synchronization of Windows settings.</p></td>
+<td align="left"><p>Select which Windows settings synchronize between computers.</p>
+<p>By default, Windows themes, desktop settings, and Ease of Access settings synchronize settings between computers of the same operating system version.</p></td>
+</tr>
+<tr class="odd">
 <td align="left"><p>Tray Icon</p></td>
 <td align="left"><p>Computers Only</p></td>
 <td align="left"><p>This Group Policy setting enables the User Experience Virtualization (UE-V) tray icon.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:121]:
```suggestion
<td align="left"><p>This Group Policy setting enables the UE-V tray icon.</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 5]
```diff
@@ -110,17 +109,30 @@ The following policy settings can be configured for UE-V.
...
 <td align="left"><p>The default is enabled.</p></td>
 </tr>
-<tr class="odd">
+<tr class="even">
 <td align="left"><p>Use User Experience Virtualization (UE-V)</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:125]:
```suggestion
<td align="left"><p>Use UE-V</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 6]
```diff
@@ -110,17 +109,30 @@ The following policy settings can be configured for UE-V.
...
 <td align="left"><p>Computers and Users</p></td>
 <td align="left"><p>This Group Policy setting lets you enable or disable User Experience Virtualization (UE-V).</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:127]:
```suggestion
<td align="left"><p>This Group Policy setting lets you enable or disable UE-V.</p></td>
```

File: mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md [hunk 7]
```diff
@@ -110,17 +109,30 @@ The following policy settings can be configured for UE-V.
...
 <td align="left"><p>Enable or disable this Group Policy setting.</p></td>
 </tr>
+<tr class="odd">
+<td align="left"><p>VDI Configuration</p></td>
+<td align="left"><p>Computers and Users</p></td>
+<td align="left"><p>This policy setting configures the synchronization of UE-V rollback information for computers running in a pooled VDI environment. If policy is enabled, the UE-V rollback state is copied to the settings storage location on logout and restored on login.</p></td>
```
Comment (JohanFreelancer9) [inline mdop/uev-v2/configuring-ue-v-2x-with-group-policy-objects-both-uevv2.md:133]:
```suggestion
<td align="left"><p>This policy setting configures the synchronization of UE-V rollback information for computers running in a pooled VDI environment. If the policy is enabled, the UE-V rollback state is copied to the settings storage location on logout and restored on login.</p></td>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
This PR seems a bit stale. Any chance of having this table updated soon, and get the linked old issue closed?

**Pull request merged**